.nvmrc

```diff
@@ -1 +1 @@
-8.9.4
\ No newline at end of file
+v8.16.0
```

Comment:
Is there a specific reason why we need to update node 8? We've done a lot of testing on Node 8.9.4 and we're going to have to do a major node update later this year anyway, so I'd rather not change this unless there's a significant tooling advantage.

Reply:
This is mainly a convenience.
Since we're stuck with a legacy version I found it best to move to a long term support version for less errors. I'm guessing this is used mainly by the electron app for wallet  handling?

Reply:
I thought Node 8.9.4 was still an LTS release, is it?

Reply:
Interesting! So I've learned that Node 8 entered LTS with [8.9.0](https://nodejs.org/en/download/releases/). It appears that lts/carbon covers 8.9.0 till 8.16.0.
Node uses an [even numbered vs an odd numbered](https://nodesource.com/blog/understanding-how-node-js-release-lines-work/) cycle where 
> you can effectively equate even-numbered releases with LTS release lines, and odd-numbered releases with non-LTS release lines.

So in fact they are both LTS. I initially thought it was only 8.16.0 since it's the version that nvm proposes for lts/carbon.

Looking at the [changelog](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V8.md) for Node 8, it seems that a lot of the minor version changes are security fixes. Combined with the ease of setting `nvm use lts/carbon` I still suggest that we bump the version.

If you feel otherwise, let me know and I'll revert the change.